.gitattributes

```diff
@@ -192,7 +192,7 @@ tools/ocaml-objcopy-macosx text eol=lf
 tools/ocamlsize text eol=lf
 tools/pre-commit-githook text eol=lf
 tools/markdown-add-pr-links.sh text eol=lf
-runtime/caml/sizeclasses.h typo.missing-header typo.white-at-eol
```

Comment:
Why has `typo.white-at-eol` been removed?

Reply:
It turned out we didn't have whitespace at eol if I regenerated the size classes and so typo complained that it was present.